.evergreen/config.yml

```diff
@@ -585,6 +585,19 @@ tasks:
           vars:
             TESTS: "tests/atlas.phpt"
 
+    - name: "test-requireApiVersion"
+      tags: ["versioned_api"]
+      commands:
+        - func: "compile driver"
+        - func: "bootstrap mongo-orchestration"
+          vars:
+            TOPOLOGY: "server"
+            AUTH: "auth"
```

Comment:
Is this related to https://github.com/mongodb/mongo-php-driver/pull/1206?

Reply:
Yes, this is how I discovered this. I'll rebase this PR once the original has merged, which will also take care of the remaining test failures.